### DISCUSSION TITLE
Add customization to Panel width

### DISCUSSION THREAD

**Discussion by Kryozen** [linuxmint/discussions#216] [Ideas]
Hi!
I opened a thread on the Mint forum to achieve what I'm trying to do but I've been told that this feature needs to be added by the developers.
I have my bottom Panel that displays the applications in the middle of it. The Applets are all in my top panel, so there is a lot of empty space in the right and left side of the panel. I'd like to customize my panel such that the empty space is not displayed as black rectangle, but the panel width adjusts to the content. I tried to do so using the .css file but I failed to achieve my aim. 

I tried to look for some related discussions but I couldn't find any.

**Comment by 100savage** [comment]
Have you tried to make the panel transparent? You can find the app in system settings/extensions.
I know this isn't the same, but it might be an easy fix for you.

**Reply by Kryozen** [reply]
Sorry for never answering! I tried your suggestion but it didn't work as I intended. I forgot to reply! But thanks for the suggestion

**Comment by Logansfury** [comment]
I have seen this feature in Xfce and I believe it would be a great addition to Cinnamon. I would enjoy it and I am sure many more users would as well.

**Comment by anaximeno** [comment]
This probably could be added as an extension of something like that, if someone wants to.

**Reply by Logansfury** [reply]
Knowing the Extension "transparent panels" works to effect the panels appearance, I was thinking it would most likely be an extension, but I am not a coder and am not aware if it is equally possible with applet or desklet on Cinnamon.

Do many coders follow this forum? If anyone capable could create this feature by any means possible it would be amazing :)
